src/js/overlay.js

```diff
@@ -201,8 +201,6 @@ Overlay.prototype.show = function() {
 		document.body.appendChild(shadow);
 	}
 
-	this.content.focus();
```

Comment:
You know a lot more about accessibility than me, so I will definitely speak
with you about that

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.